tests/unit/test-es.js

```diff
@@ -22,17 +22,6 @@ test('search id parameter doesnt override other parameters', async (t) => {
   )
 })
 
-test('invalid search datetime parameter intervals fail', async (t) => {
```

Comment:
this is unnecessary, as all of the datetime parameter validation is done prior to this method, so the es client doesn't do it again.